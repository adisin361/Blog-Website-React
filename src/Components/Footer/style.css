.main-footer {
  background-color: black;
  color: aliceblue;
  font-family: Arial, Helvetica, sans-serif;
  height: 10vh;
  padding-left: 19vw;
  display: flex;
  align-items: center;
}