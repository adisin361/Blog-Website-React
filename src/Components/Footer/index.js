import * as React from 'react';
import './style.css';
const Footer = () => {
  return (
    <>
      <div className='main-footer'>
        <p>&copy;artifact.com 2019</p>
      </div>
    </>
  );
};

export default Footer;