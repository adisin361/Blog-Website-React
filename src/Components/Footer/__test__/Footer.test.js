import "@testing-library/jest-dom";
import { fireEvent, render } from "@testing-library/react";
import React from "react";
import Footer from '..';

describe("Footer", () => {
  it("should match the snapshot", () => {
    const { asFragment } = render(<Footer />);
    expect(asFragment()).toMatchSnapshot();
  });
});