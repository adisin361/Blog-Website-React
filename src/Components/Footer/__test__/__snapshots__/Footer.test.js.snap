// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should match the snapshot 1`] = `
<DocumentFragment>
  <div
    class="main-footer"
  >
    <p>
      Â©artifact.com 2019
    </p>
  </div>
</DocumentFragment>
`;
